lib/cext/preprocess.rb

```diff
@@ -145,6 +180,63 @@
         replacement: '\1->body = rb_tr_handle_for_managed(\2);'
       }
     ]
+  },
+  'client.c' => {
+    gem: 'mysql2',
+    patches: [
+        read_field('wrapper','active_thread'),
+        write_field('wrapper','active_thread', false),
+        read_field('wrapper','encoding'),
+        write_field('wrapper','encoding', false),
+        read_field('async_args','self'),
+        write_field('async_args','self', true),
+        read_field('args','sql'),
+        write_field('args','sql', true),
```

Comment:
These could be un-indented by 2 spaces.
It might be nice to create a helper doing read_field + write_field returning an Array.

Reply:
I've unindented these and updated to `read_write_field` method.